playwright/playwright.config.ts

```diff
@@ -0,0 +1,29 @@
+import { PlaywrightTestConfig, devices } from '@playwright/test';
+
+const chromium = 'chromium' as const;
+const allSupportedBrowsers = [chromium, 'firefox', 'webkit'] as const;
+const mobileDeviceNames = ['iPhone 11'] as const;
```

Comment:
Did you pick these, or is this based on a list of supported browsers we have?

(I know Gareth E was planning to review the list, so it'd be good to have them linked and consistent if so)